packages/web-runtime/src/composables/upload/useUpload.ts

```diff
@@ -9,7 +9,6 @@ import { computed, Ref, unref, watch } from '@vue/composition-api'
 import { useActiveLocation } from 'files/src/composables'
 import { isLocationPublicActive } from 'files/src/router'
 import { UppyService } from '../../services/uppyService'
-import { Location } from 'vue-router/types/router'
 import * as uuid from 'uuid'
 
 export interface UppyResource {
```

Comment:
Could you please add a comment that this interface MUST never contain complex types / MUST always consist of primitive types only? With the serialization reasoning. I guess that's a fair warning that should stay in the code to not run into the same trap again in the future.